x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/cardinality.tsx

```diff
@@ -112,21 +112,17 @@ export const cardinalityOperation: OperationDefinition<CardinalityIndexPatternCo
   documentation: {
     section: 'elasticsearch',
     signature: i18n.translate('xpack.lens.indexPattern.cardinality.signature', {
-      defaultMessage: 'field: string, [kql]?: string, [lucene]?: string',
+      defaultMessage: 'field: string',
     }),
     description: i18n.translate('xpack.lens.indexPattern.cardinality.documentation', {
       defaultMessage: `
 Calculates the number of unique values of a specified field. Works for number, string, date and boolean values.
 
 Example: Calculate the number of different products:
-\`\`\`
-unique_count(product.name)
-\`\`\`
+${'`unique_count(product.name)`'}
 
 Example: Calculate the number of different products from the "clothes" group:
-\`\`\`
-unique_count(product.name, kql="product.group=clothes")
-\`\`\`
+${'`unique_count(product.name, kql="product.group=clothes")`'}
```

Comment:
Since you're making a change to kql here, can you also ensure that KQL uses single quotes in every example?